collector/cpu_openbsd.go

```diff
@@ -56,18 +56,18 @@ func (c *cpuCollector) Update(ch chan<- prometheus.Metric) (err error) {
 		return err
 	}
 
-	var cp_time [][C.CPUSTATES]C.int64_t
+	var cp [][C.CPUSTATES]C.int64_t
```

Comment:
This could also be `cpTime`.

Reply:
Fine for me, I've just pushed a new patch.